source/request-response/request_response_client.c

```diff
@@ -92,22 +105,17 @@ Client Tables/Lookups
     (Authoritative operation container)
```

Comment:
Trivial: This comment block is super helpful for quick understanding of the 4 current hash tables. They're easily identifiable but could we still label these with the names used for the members in the rr client below?

Reply:
Updated in FinalPolish PR